portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsSingleSelectionField.testcase

```diff
@@ -123,13 +123,66 @@ definition {
 	}
 
 	@description = "Verify that a Help Text can be added"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test HelpTextCanBeAdded {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-132199 HelpTextCanBeAdded pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+            fieldPositionNumber = "0",
+            fieldType = "Single Selection",
+            formPageNumber = "0");
+
+		var fieldSingleSelectionField = FormFields.getFieldName();
+
+		Type(
+			key_rowNumber = "1",
+			locator1 = "FormFields#OPTIONS_OPTION_VALUE_FIELD",
+			value1 = "Option 1");
+
+		Type(
+			key_rowNumber = "2",
+			locator1 = "FormFields#OPTIONS_OPTION_VALUE_FIELD",
+			value1 = "Option 2");
+
+		FormViewBuilder.setFieldHelpText(fieldLabel = "Single Selection Help Text");
+
+		Form.save();
+
+		Form.publishForm();
+
+		FormsAdminNavigator.gotoPublishCopyURLLink();
+
+		var publishedFormPublicURL = Form.getPublishedFormPublicURL(
+			publishedFormPublicURL = "${publishedFormPublicURL}");
+
+		Navigator.openSpecificURL(url = "${publishedFormPublicURL}");
```

Comment:
There is a macro called `FormsAdminNavigator.gotoPublishedForm()` that performs those steps which are declared in the testcase. May I ask you to replace the lines 162 to 167 with this macro?

And to improve it and avoid any flakyness, may I ask you to also add `FormPortlet.waitFormRenderer(text = "Submit")`? It will ensure that this test is only going to move ahead once the Submit is rendered on the form page.

It will look like this:

```
FormsAdminNavigator.gotoPublishedForm();

FormPortlet.waitFormRenderer(text = "Submit");
```